src/helpers/fips.ts

```diff
@@ -0,0 +1,16 @@
+import crypto from 'crypto'
+
+export const enableFips = (fips: boolean, ignoreError = false) => {
+  try {
+    // FIX-ME The mark command is calling tag.execute function, and when doing so, the fips option
+    // is not reduced to a boolean, but remains a clipanion option, mistakenly enabling fips.
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-boolean-literal-compare
```

Comment:
The current approach to fix this is to use setters (like [explained here](https://github.com/DataDog/datadog-ci/blob/70afd0ecee4933251b045cb90c71edeb02a394a5/src/commands/tag/tag.ts#L50-L56)).

Maybe in the future we'll want some "reset" function that takes the `this` of the parent command, to inherit its reduced options.

But let's not do that in your PR 👌

Reply:
My understanding is that this is caused by the confusion between the command and the functions which are executed when calling the command.

We would have two valid way of calling a command from another one:
- calling the command is done through CLI, or through `this.cli.run(['my-command', ...arguments])`" documented [here](https://mael.dev/clipanion/docs/api/cli#clirun)
- calling the function behind the command is done entirely in js, and providing a valid config object, e.g. `sendTags(ciEnv, level, provider, tags)`.

But instead of these two valid approaches, we are using an in-between, which can't initialise the command properly because it's not wrapped by Clipanion runtime, and it can't pass the expected options either, since it's not the plain js function expecting arguments.

Reply:
Indeed, both would be better Clipanion-y ways to do it 😁